tests/test-dirs/file-cache.t/run.t

```diff
@@ -0,0 +1,66 @@
+The server might already be running, we kill it to make sure we start from a
```

Comment:
This test was removed on master by 4885781c361177dd93b96deb35d13be94d6e38a0
I guess you missed that in the rebase and that's why you're reintroducing it. But it could be deleted.